packages/react-devtools-extensions/src/SourceMapMetadataConsumer.js

```diff
@@ -15,7 +15,7 @@ import type {
   MixedSourceMap,
 } from './SourceMapTypes';
 import type {HookMap} from './generateHookMap';
-import * as util from 'source-map/lib/util';
+import * as util from 'source-map-js/lib/util';
 import {decodeHookMap} from './generateHookMap';
 import {getHookNameForLocation} from './getHookNameForLocation';
 
```

Comment:
(for some reason can't leave comments below this line, but we should also update the comment in line 30 reflecting that we are no longer using the source-map library